src/TurbulenceClosures/turbulence_closure_implementations/TKEBasedVerticalDiffusivities/catke_vertical_diffusivity.jl

```diff
@@ -70,28 +70,42 @@ Arguments
 
 - `FT`: Float type; default `Float64`.
 
-
 Keyword arguments
 =================
 
-- `maximum_diffusivity`: Maximum value for tracer, momentum, and TKE diffusivities.
-                         Used to clip the diffusivity when/if CATKE predicts
-                         diffusivities that are too large.
-                         Default: `Inf`.
+- `mixing_length`: The formulation for mixing length; default: `CATKEMixingLength()`.
+
+- `turbulent_kinetic_energy_equation`: The TKE equation; default: `CATKEEquation()`.
+
+- `maximum_tracer_diffusivity`: Maximum value for tracer diffusivity. CATKE-predicted tracer diffusivities that are larger
+                                than `maximum_tracer_diffusivity` are clipped. Default: `Inf`.
+
+- `maximum_tke_diffusivity`: Maximum value for TKE diffusivity. CATKE-predicted diffusivities for TKE that are larger
+                             than `maximum_tke_diffusivity` are clipped. Default: `Inf`.
+
+- `maximum_viscosity`: Maximum value for momentum diffusivity. CATKE-predicted momentum diffusivities that are larger
+                       than `maximum_viscosity` are clipped. Default: `Inf`.
 
 - `minimum_tke`: Minimum value for the turbulent kinetic energy.
                  Can be used to model the presence "background" TKE
                  levels due to, for example, mixing by breaking internal waves.
-                 Default: 0.
+                 Default: 1e-9.
+
+- `minimum_convective_buoyancy_flux` Minimum value for the convective buoyancy flux. Default: 1e-11.
 
 - `negative_tke_damping_time_scale`: Damping time-scale for spurious negative values of TKE,
                                      typically generated by oscillatory errors associated
-                                     with TKE advection.
-                                     Default: 1 minute.
+                                     with the TKE advection. Default: 1 minute.
+
+!!! note "Numerical stability"
+    For numerical stability, it is recommended to either have a relative short
+    `negative_tke_damping_time_scale` or a reasonable `minimum_tke`, or both.
```

Comment:
It's hard for me to gauge what "relatively short" or "reasonable" time steps are. Could we help the users here?